content/core-features/channels.textile

```diff
@@ -0,0 +1,31 @@
+---
+title: Channels
+section: core-features
+index: 23
+---
+
+Ably aggregates all it's data into named units of distribution, referred to as "channels". Channels offer a way to implement the Publish-Subscribe (Pub/Sub) architectural pattern, which is a popular pattern used for realtime data delivery. 
+
+The Publish-Subscribe messaging pattern lets any number of publishers publish data to a channel, which could be subscribed to by any number of subscribers. The key thing to note about Pub/Sub is that publishers and subscribers are completely decoupled as explained further. Besides, once subscribed, the subscribers no longer have to poll the server or data provider to check if there is any new data that they need to be aware of; instead, they will be notified of it as it becomes available.
+
+h2(#understanding-pubsub). Understanding Pub/Sub with an example
+
+To understand Publish-Subscribe in more detail, let's consider an example of location tracking of a vehicle in realtime. In this case, the vehicle whose location is to be tracked acts as a publisher, while the user intending to receive the location updates acts as a subscriber. This scenario is illustrated below.
```

Comment:
At this stage we've introduced that `Publish-Subscribe` is `Pub-Sub`, so it'd be good if we were consistent in using one or the other.